src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,270 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: sa
+      changes:
+        - createSequence:
+            sequenceName: id_seq
+            incrementBy: 1
+            startValue: 1
+            
+  - changeSet:
+      id: 2
+      author: sa
+      changes:
+        - createSequence:
+            sequenceName: proc_type_id_seq
+            incrementBy: 1
+            startValue: 1          
+  - changeSet:
+      id: 3
+      author: sa
+      changes:
+        - createTable:
+            tableName: PROCESS_TYPE
+            columns:
+              - column:
+                  name: ID
+                  type: long
+                  autoIncrement: true
+
+              - column:
+                  name: NAME
+                  type: varchar(50)
+
+              - column:
+                  name: LABEL
+                  type: varchar(50)
+  -  changeSet:
+      id: 4
+      author: sa
+      changes:
+        - addPrimaryKey:
+            columnNames: ID
+            constraintName: pk_proc_type_id
+            tableName: PROCESS_TYPE
+ 
+  - changeSet:
+      id: 5
+      author: sa
+      changes:
+        - createTable:
+            tableName: ACTION_REQUEST
+            columns:
+              - column:
+                  name: ID
+                  type: long
+                  autoIncrement: true
+
+              - column:
+                  name: PROCESS_INSTANCE_ID
+                  type: varchar(50)
+
+              - column:
+                  name: TITLE
+                  type: varchar(50)
+                  
+              - column:
+                  name: PROCESS_TYPE_ID
+                  type: long
+
+              - column:
+                  name: PROCESS_INSTANCE_STATUS
+                  type: varchar(100)
+                  
+              - column:
+                  name: ACTION_REQUESTED
+                  type: varchar(100)
+
+              - column:
+                  name: INITIATOR
+                  type: varchar(100)
+
+              - column:
+                  name: DELEGATOR
+                  type: varchar(100)
+              
+              - column:
+                  name: GROUP_
+                  type: varchar(100)
+                  
+              - column:
+                  name: ROUTE_NODES
+                  type: varchar(100)
+
+              - column:
+                  name: LAST_APPROVED_DATE
+                  type: varchar(100)
+
+              - column:
+                  name: CREATION_DATE
+                  type: varchar(100)
+                  
+  -  changeSet:
+      id: 6
+      author: sa
+      changes:
+        - addPrimaryKey:
+            columnNames: ID
+            constraintName: pk_check_req_id
+            tableName: ACTION_REQUEST
+
+  -  changeSet:
+      id: 7
+      author: sa
+      changes:
+        - addForeignKeyConstraint:
```

Comment:
You can have multiple changes in a changeset (so you can `createTable` and `addForeignKeyConstraint` in the same change set) and if they're logically grouped together (and haven't already been run on a persistent database), they should go together.